### ISSUE TITLE
UI - Upcoming Lessons Buttons - Need Correct Hover and Selected State

### ISSUE THREAD

**Issue by sethborne** [RecursiveThinking/recursive_thinking_website#67]
I know we closed #17 earlier - but I think Kennon is still working on these two things:

- correct hover state (darker background/border)

- correct "selected" state, where selected button is original state, the other two are "grayed out".

**Comment by MynockSpit** [comment]
Oh, did I close it too early? Whoops! Feel free to reopen it if I did so mistakenly.

**Comment by sethborne** [comment]
not at all, I think you closed it at a perfect time.  the old issue was about "selectability", which Kennon solved.

But I know Kennon talked to Hailey (last Sat) about how exactly she saw both the hover, and the selected/non-selected state occurring, and I think he is still working on both of those ATM.

**Comment by kennh** [comment]
I was working on other issues after talking to Haley about the function of the buttons.

**Comment by sethborne** [comment]
@kennh if my assessment of the next steps for the hover buttons are incorrect, please let me know

**Comment by kennh** [comment]
Your assessment of the next steps are what we talked about and should be correct.
